calendar/store/test_util.go

```diff
@@ -0,0 +1,27 @@
+package store
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/calendar/testutil"
+	"github.com/mattermost/mattermost-plugin-mscalendar/calendar/tracker/mock_tracker"
+	"github.com/mattermost/mattermost-plugin-mscalendar/calendar/utils/bot/mock_bot"
+)
+
+var MockString = mock.AnythingOfType("string")
```

Comment:
Can you put the matched by here as well.  Eventually we will move these into a util file and re-use.